packages/decoder/package.json

```diff
@@ -21,7 +21,7 @@
     "build": "tsc",
     "prepare": "yarn build",
     "start": "tsc --watch",
-    "test": "mocha ./test/current/test/* ./test/legacy/test/*"
+    "test": "mocha ./test/current/test/* ./test/legacy/test/* --exit"
```

Comment:
Can we make sure this isn't necessary when using the decoder as a library? (i.e., not via tests)

Reply:
Is there a missing `provider.disconnect()` somewhere in there?

Reply:
There were missing `await providerr.disconnect()` statements, as well as our tests using the incorrect Ganache instamine mode! Thanks to @MicaiahReid for the assist on tracking this down.